install/kubernetes/helm/meshery-operator/crds/crds.yaml

```diff
@@ -36,6 +36,7 @@ spec:
             description: BrokerSpec defines the desired state of Broker
             properties:
               size:
+                description: Desired size of the Broker, will be an integer value denoting the number of Broker instances that should be in one cluster, currently it is adviced to have one Broker instance in a cluster but that can be scaled vertically up or down depending on load
```

Comment:
not vertically, horizontally.

Reply:
Just don't mention direction and we're fine.

Reply:
Done.